main.cpp

```diff
@@ -15,7 +15,9 @@ int main(int argc, char* argv[])
   //close the file right away so if the program fails anywhere else, the log file won't be left open.
   //the log file is opened anywhere it needs to be read from or written to, so it is ok to close it here.
   std::ofstream ofile("request_response_log.txt");
```

Comment:
The other writes only add to the end of file, and i just wanted to open it at the beginning no matter what, just in case someone calls the status handler before any other request have been made, so there is some file that exists (empty).